.azure_templates/t-extend-database-build.yml

```diff
@@ -0,0 +1,98 @@
+parameters:
+  - name: containerRegistry
+    default: "utPLSQL-docker"
+  - name: dockerRepo
+    default: "utplsqlv3/oracledb"

```

Comment:
Can we switch to the public docker repo from @gvenzl
It will make this template reusable as our docker images repo is private and requires credentials to access it.
`gvenzl/oracle-xe:21-slim`

Reply:
So this is a just default value. If you notice the value for that parameter is coming from main template `t-extend-database-build.yml`  and can be passed as parameter to the template. So all this values are in fact parameters that you can pass.
These are not hardcoded apart from default if you dont pass so for example you would pass :
```yml
 - template: .azure_templates/t-extend-database-build.yml
    parameters:
      containerRegistry: YourRegistry
      dockerRepo: YourRepo
      utplsqlVersion: ${UTPLSQL_VERSION}
      utplsqlDir: ${UTPLSQL_DIR}
      test_user: ${DB_USER}
      test_pwd: ${DB_PASS}
      targetDatabases:
```
However I will update default to gvenzl docker hub repo.
Now, containerRegistry in my opinion is misleading name used by Azure to describe your serviceConnection defined for container registry...